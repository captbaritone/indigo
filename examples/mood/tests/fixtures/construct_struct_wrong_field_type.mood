struct Foo {
  a: i32,
  b: i32
}

pub fn test(): i32 {
    Foo { a: 1_i32, b: 1_f64 };
    0_i32
}