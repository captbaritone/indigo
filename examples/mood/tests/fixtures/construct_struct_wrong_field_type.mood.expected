Error: Missing struct field(s): "a", "b".:
 --> construct_struct_wrong_field_type.mood:7:30

  | pub fn test(): i32 {
7 |     Foo { a: 1_i32, b: 1_f64 };
  |                              ^ "Foo" is missing the field(s) "a", "b".
  |     0_i32
